portfolio/src/main/webapp/about.css

```diff
@@ -0,0 +1,42 @@
+.font {
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
nit: Indentation should be 2 spaces (most of this file is good but a few sections deviate): https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
fixed! Always forget that it does 4 spaces by default and that I have to change it.